api/open_saves.proto

```diff
@@ -355,6 +356,11 @@ message QueryRecordsRequest {
 
   // owner_id is the owner of records, represented as an external user ID.
   string owner_id = 4;
+
+  // List of sort orders to return records. These SortOrders are applied 
+  // in sequence.
+  repeated SortOrder orders = 5;
```

Comment:
I'd suggest sort_orders to be consistent with other fields (MessageName message_name).